docs/30-cloud-storage.md

```diff
@@ -5,13 +5,9 @@ Cloud stoarge for spark enables you to have a persisted storage system backed by
 
 Pre-built into this package is native support for connecting your Spark cluster to Azure Blob Storage (aka WASB). The required WASB jars are automatically placed in the spark cluster and the permissions are pulled from you secrests file.
 
-To connect to your Azure Storage account, make sure that the storage fields in your *.aztk/secrets.yaml* file are properly filled out.
+To connect to your Azure Storage account, make sure that the storage fields in your *.aztk/core-site.xml* file are properly filled out.
 
-[Even if you are just testing and have no need to connect with Azure Blob Storage, you still need to correctly fill out the storage fields in your *.aztk/secrets.yaml* file as it is a requirement for this package.]
```

Comment:
i would leave in this sentiment - people may be wondering why they need to put storage creds in the secrets.yaml file if they are using ADLS.

Reply:
good point. will add back

Reply:
Actually.. I'm not sure how it reads... since we've removed the WASB stuff from there as well I think it is very strange to have this in here.